main.py

```diff
@@ -948,6 +954,16 @@ def notify_switch_enabled(self, dpid):
         event = KytosEvent(name=name, content={'dpid': dpid})
         self.controller.buffers.app.put(event)
 
+    def notify_switch_links_status(self, switch, reason):
+        """Send an event to notify the status of a link in a switch"""
+        with self._links_lock:
+            for link in self.links.values():
+                if switch in (link.endpoint_a.switch, link.endpoint_b.switch):
+                    if reason == "link enable":
+                        self.notify_link_up_if_status(link, reason)
```

Comment:
@Alopalao, since we have the `time.sleep` when enabling a link calling this from a request handler will increase and hold the response time, could you please refactor to add to new method to dispatch the call to `notify_link_up_if_status(link, reason)` via an internal event, maybe `kytos/topology.notify_link_up_if_status`?  and then instead of calling directly `self.notify_link_up_if_status` you call this new method that will publish the event accordingly to `kytos/topology.notify_link_up_if_status`  and then you implement a new event handler for internal usage?

By convention, we typically name `on_*` methods that handle events

Reply:
Good thing that the request handler is also threaded but increasing the response time is a blocker for us :)